LOG.tbasicu

```diff
@@ -103,10 +104,10 @@ Type tLogThis
     me.isDebug = %FALSE
     me.verbose = %FALSE
     me.prependTag = %TRUE
-    me.ColorTime = %CONSOLE_FOREGROUND_BLUE + %Console_FOREGROUND_GREEN + %CONSOLE_FOREGROUND_INTENSITY ' bright cyan
```

Comment:
There were both + and OR, unified to OR.

Reply:
ok!